2019/01/14.md

```diff
@@ -5,18 +5,18 @@
 - Nicolò
 - Sven
 - Henry
-- Kai
+- Kai (babel-eslint, ESLint)
 - Logan
-- Denis
-- Fransisco
+- Denis ([core-js](https://github.com/zloirock/core-js))
+- Fransisco ([RunKit](https://runkit.com/home))
 
 ## Do we want Babel to be compatible with ESTree ASTs
 
 > [babel/babel#9253](https://github.com/babel/babel/pull/9253), [babel/babel#9258](https://github.com/babel/babel/pull/9258)
```

Comment:
Part of this conclusion should mean we should close the PRs/issues and write up a statement on our viewpoint regarding this with more detail. Something like "ideally yes" but no because xyz. Not sure where we would put it, I forgot where we wrote down these kinds of statements like vision etc.